src/olympia/reviewers/utils.py

```diff
@@ -969,10 +969,12 @@ def set_file(self, status, file):
         file.datestatuschanged = datetime.now()
         if status == amo.STATUS_APPROVED:
             file.approval_date = datetime.now()
-        file.status = status
         if status == amo.STATUS_DISABLED:
+            file.original_status = file.status
+        else:
             file.original_status = amo.STATUS_NULL
-            file.status_disabled_reason = File.STATUS_DISABLED_REASONS.NONE
+        file.status_disabled_reason = File.STATUS_DISABLED_REASONS.NONE
```

Comment:
I figured any time we're updating the status of a file we want to clear `status_disabled_reason` (?)